### PR TITLE
Add new guide for payment flows

### PR DESCRIPTION
**Pull request by IsaacMurtagh**
image
image

Test plan: Navigate to the page and expect the guide is there.

### CODE REVIEW

**Comment by IsaacMurtagh** [comment]
WIP: while i await sign off on content

File: src/content/guides/payment-flows.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+---
+Transacting via Centrapay requires customer and merchant devices to negotiate the creation and fulfillment of Payment Requests. There are multiple payment “flows” a merchant integration can select from to facilitate this negotiation. Which flow is employed will depend on the capabilities of the merchant system.
+
+## Merchant-presented
+
+When a merchant terminal has a customer-facing display it can prompt the customer to pay by showing a QR code.
+
+### Dynamic Merchant QR Code
+
+1. Merchant terminal [creates a payment request](https://docs.centrapay.com/api/payment-requests#create-a-payment-request) via Centrapay API and shows QR code on customer-facing display.
```
Comment (laurencemcknight) [inline src/content/guides/payment-flows.mdx:16]:
```suggestion
1. Merchant terminal [creates a payment request](https://docs.centrapay.com/api/payment-requests#create-a-payment-request) via Centrapay API and shows the QR code on customer-facing display.
```

File: src/content/guides/payment-flows.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+title: Payment Flows
+description: Centrapay facilitates secure and convenient transactions between customer and merchant devices, with multiple payment flows available for selection. Transact with ease and confidence using Centrapay.
+nav:
+  path: Reference/Centrapay Experiences
+  order: 1
+---
+Transacting via Centrapay requires customer and merchant devices to negotiate the creation and fulfillment of Payment Requests. There are multiple payment “flows” a merchant integration can select from to facilitate this negotiation. Which flow is employed will depend on the capabilities of the merchant system.
```
Comment (laurencemcknight) [inline src/content/guides/payment-flows.mdx:8]:
```suggestion
Transacting via Centrapay requires customer and merchant devices to negotiate the creation and fulfilment of [Payment Requests](https://docs.centrapay.com/api/payment-requests). There are multiple payment “flows” a merchant integration can select from to facilitate this negotiation. Which flow is employed will depend on the capabilities of the merchant system.
```

**Review by laurencemcknight — APPROVED** [review]
Mermaid diagram looks good 👍

**Pull request merged**